plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorOutlinePage.java

```diff
@@ -1003,6 +1010,113 @@ public Object visitSelectColumnSpec(@NotNull ColumnSpec columnSpec, @NotNull Out
             this.makeNode(arg, columnSpec, text, extraText, DBIcon.TREE_COLUMN, columnSpec.getValueExpression());
             return null;
         }
+        
+
+        @Nullable
+        @Override
+        public Object visitTableStatementDelete(@NotNull SQLQueryTableDeleteModel deleteStatement, @NotNull OutlineQueryNode node) {
+            String tableName = deleteStatement.getTableModel() == null ? "?" : deleteStatement.getTableModel().getName().toIdentifierString();
```

Comment:
Maybe we can add a constant for the "?" symbol? We have 3 in this PR and two more in the ui.editors.sql

Reply:
Yes, I wanted to do it in scope of refactoring ticket https://github.com/dbeaver/pro/issues/2529, but if you think that it's better to do now, okay then